src/PatientChecking/PatientChecking/Views/Shared/_Layout.cshtml

```diff
@@ -30,7 +30,7 @@
                         Patients
                         <span class="fas fa-caret-down first" ></span>
                     </a>
-                    <ul class="@(@ViewBag.Controller == "Patient" ? "patient-show show" : "patient-show")" >
+                    <ul class="@(@ViewBag.Controller == "Patient" ? "patient-show show" : "patient-show")" style="padding-left:0px">
```

Comment:
inline css was told you in last reviews but I see it still apply here.
If you follow Bootstrap docs,
https://getbootstrap.com/docs/5.0/utilities/spacing/
using style="padding-left:0px" is similar to adding class "ps-0".

Reply:
I will fix this in the new branch based on develop and fix layout as per reviewer request

Reply:
I have replaced in-line style with bootstrap class